lib/openssl/config.rb

```diff
@@ -53,9 +53,8 @@ def parse(string)
       def parse_config(io)
         begin
           parse_config_lines(io)
-        rescue ConfigError => e
-          e.message.replace("error in line #{io.lineno}: " + e.message)
-          raise
+        rescue => error
```

Comment:
It should probably still catch only `ConfigError`

Reply:
It can raise another new exception.
```ruby
          raise e.class, "error in line #{io.lineno}: #{e.message}", e.backtrace
```